PennMobile/Dining/Networking + Cache/StorageType.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  StorageType.swift
+//  PennMobile
+//
+//  Created by Dominic Holmes on 10/21/19.
+//  Copyright © 2019 PennLabs. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - Data Store Types
+enum DataStoreType {
+    case cache // Can be cleared by system
+    case permanant // Never cleared
+    
+    var searchPathDirectory: FileManager.SearchPathDirectory {
+        switch self {
+        case .cache: return .cachesDirectory
+        case .permanant: return .documentDirectory
+        }
+    }
+    
+    var folder: URL {
+        let path = NSSearchPathForDirectoriesInDomains(searchPathDirectory, .userDomainMask, true).first!
```

Comment:
Any risk of this failing? I'm generally not a fan of bang operators.

Reply:
got rid of it ^ unclear if this could fail